metadata-ingestion/docs/sources/powerbi/powerbi_pre.md

```diff
@@ -100,6 +100,8 @@ combine_result
 
 `Pattern-1` is supported as it first assign the table from schema to variable and then variable is used in M-Query Table function i.e. Table.Combine
 
+Note: If you are using (PowerBI Parameters)[https://learn.microsoft.com/en-us/power-query/power-query-query-parameters]. then avoid using same name for variable in M-Query
```

Comment:
we should also add the caveat that we don't support parameters with the admin API

Reply:
Added it in `Admin Ingestion vs. Basic Ingestion`